client/lib/wporg/index.js

```diff
@@ -148,3 +149,14 @@ export function fetchTranslationsList( wpVersion ) {
 	const query = { version: wpVersion };
 	return getRequest( WPORG_CORE_TRANSLATIONS_ENDPOINT, query );
 }
+
+/**
+ * Get the WP.org versions.
+ * See: https://codex.wordpress.org/WordPress.org_API
+ *
+ * @returns {Promise<{ offers: [ { current: string } ] }>}  A promise that returns an object containing
+ * 								an `offers` array with the WP versions
+ */
+export function fetchWordPressVersions() {
+	return getRequest( WPORG_CORE_VERSIONS_ENDPOINT );
```

Comment:
Can we just return the versions instead of the `offers` here? `offers` seems .org specific and we can avoid using it in `useLatestWPVersions`.
```suggestion
    const WPVersions = getRequest( WPORG_CORE_VERSIONS_ENDPOINT );
	return WPVersions?.offers;
```

Reply:
In this case, returning specific `org` responses from a `wporg` package is expected to me, also the other requests in the file do not apply any transformation but I can see your point of avoiding the mapping in the consuming code so I will go ahead and change it. The important thing is not duplicating the parsing of the response, so as long as it is in one place, it is fine for me 😄

Reply:
I've now realized that the suggestion would not work as `getRequest` returns a Promise and not an object so we should wait for the Promise in the API code. I don't think this is coherent with existing code. @cpapazoglou do you strongly feel we should change that?

Reply:
Thanks for exploring our options, let's move on with the current code. It's not that important.